### DISCUSSION TITLE
MQA compliance

### DISCUSSION THREAD

**Discussion by gmartirano** [gopeg/issues-barriers#26] [General]
As documented [here](https://data.europa.eu/mqa/methodology?locale=en), MQA is based on “use of non-obligatory fields” in DCAT-AP.
Use of non obligatory DCAT-AP fields may be a problem for those metadata conformant to ISO 19115/INSPIRE standard/profile published in geodata catalogues.
The current situation for the GO-DEPTH metadata is the following:
The (xml) metadata is published in the National Geodata Catalogue, which publishes only metadata conformant to RNDT (national MD profile - based on ISO 19115 MD standard – which extends INSPIRE MD profile).
Because the dataset is also open, it is discoverable also in the National Open Data Catalogue. To make this happening, according to Italian regulations, AgID (the Party responsible for both the catalogues):

- converts the xml metadata published in the National Geodata Catalogue in a (xml/rdf) metadata conformant to DCAT-AP-IT, which is the national MD profile – based on DCAT standard – extending DCAT-AP profile and is the mandatory MD profile for the National Open Data Catalogue,

- ingests the converted metadata into the National Open Data Catalogue.

Validating the xml/rdf metadata (compliant to DCAT-AP-IT) against the DCAT-AP schema, we get few warnings (as expected), related to the extended elements of the Italian profile.
Validating the xml/rdf metadata present in the EDP with MQA, we get a lot of “violations”.
It’s worth to highlight that the two xml/rdf metadata are different

- the one published in the National Open Data Catalogue is converted in DCAT-AP-IT by AgID from the xml metadata published in the National Geodata Catalogue,

- the one published in the EDP is converted in DCAT-AP by EDP from the xml metadata published in the National Geodata Catalogue (because, regarding open geodata, EDP harvests only the National Geodata Catalogue).

We are currently investigating how MQA actually works.
Meanwhile, for metadata conformant to ISO 19115/INSPIRE standard/profile published in national geodata catalogues it emerges the potential issue to correctly identify and properly valorise the ISO 19115/INSPIRE elements corresponding to the non obligatory DCAT-AP fields used by the MQA.

**Comment by AndersOstmanNovogit** [comment]
This issue is reported in section 4.5.1 in D4.3

The Italian procedure is more or less equal to the Swedish procedure - with the same results. But since the Swedish 19115->DCAT converter is different to the Italian one, it seems that the EDP converter is the main problem.

One solution may be that the EDP only harvest from the national data portals, already having the geospatial metadata in DCAT-AP-XX format, thereby avoiding the EDP converter.

EDP harvest both the Swedish Open Data Portal and the Swedish Geoportal, creating duplicate entries, which is another problem, sicne the different conversions gives different results.
A randomly selected Swedish example is here
MQA on metadata from the geoportal: https://data.europa.eu/data/datasets/6cfad5df-c8ba-48f1-99ac-09a901b45fca/quality?locale=sv 
MQA on metadata from the data portal: https://data.europa.eu/data/datasets/https-www-geodata-se-geodataportalen-resource-6cfad5df-c8ba-48f1-99ac-09a901b45fca/quality?locale=sv 
 
@gmartirano : Two questions. 1) When validating the DCAT-AP_IT file generated by AgID, which validator did you use? MQA? 2) Have you solved the problem? If yes, how?

**Reply by gmartirano** [reply]
@AndersOstmanNovogit:
1) no, a validator available in JoinUp
2) we are still investigating the issue

**Reply by AndersOstmanNovogit** [reply]
Now moved to D4.3 section 4.4.2
